packages/components/src/components/SquigglePlayground/LeftPlaygroundPanel/index.tsx

```diff
@@ -0,0 +1,174 @@
+import {
+  ReactNode,
+  forwardRef,
+  useEffect,
+  useImperativeHandle,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+
+import { SqValuePath } from "@quri/squiggle-lang";
+import { Bars3CenterLeftIcon, Button } from "@quri/ui";
+
+import {
+  SquiggleOutput,
+  useRunnerState,
+  useSquiggle,
+  useUncontrolledCode,
+} from "../../../lib/hooks/index.js";
+import { altKey, getErrors } from "../../../lib/utility.js";
+import { CodeEditor, CodeEditorHandle } from "../../CodeEditor.js";
+import { PlaygroundSettings } from "../../PlaygroundSettings.js";
+import { AutorunnerMenuItem } from "./AutorunnerMenuItem.js";
+import { MenuItem } from "./MenuItem.js";
+import { RunMenuItem } from "./RunMenuItem.js";
+import { SetttingsMenuItem } from "./SettingsMenuItem.js";
+
+type Override = {
```

Comment:
I don't like the name "Override", it's not specific enough. Maybe "AltPanelView" or "AltView" or similar?